en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md

```diff
@@ -12,20 +12,47 @@ Following diagram shows the overall solution we are going to build. The SMSC wil
 
 <img src="{{base_path}}/assets/img/integrate/connectors/smpp-inboundep-example.png" title="SMPP Inbound Endpoint" width="800" alt="SMPP Inbound Endpoint"/>
 
-## Configure inbound endpoint using WSO2 Integration Studio
+## Setting up the inbound endpoint using Micro Integrator
```

Comment:
Not related to this PR. But fix the first paragraph under `What you'll build` It has so many basic issues. 

Suggestion : 
```This scenario demonstrates how the SMPP inbound endpoint works as a message consumer. In this scenario, you should have connectivity with SMSC (Short Message Service Center) via the SMPP protocol. For this, we are using **SMSC simulator** to accomplish the required requirements. Please refer to the [Setting up the SMPP Connector]({{base_path}}/reference/connectors/smpp-connector/smpp-connector-configuration/) documentation for more information.```